### PR TITLE
Fixed #34618 -- Added icon descriptions in "Recent Actions" on the admin index.

### PR DESCRIPTION
**Pull request by nmenezes0**
Fixes https://code.djangoproject.com/ticket/34618

### CODE REVIEW

**Comment by DevilsAutumn** [comment]
Hi, Just to avoid linting and formatting errors you can set up pre commit checks. See [here](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#pre-commit-checks) for instructions.

**Comment by DevilsAutumn** [comment]
> @DevilsAutumn as far as I can see there are no pre-commit checks for CSS. Be great to add some?

But i can't see any linting errors related to CSS. I hope pre commit checks work in `admin_views/tests.py`. 😄

File: tests/admin_views/tests.py [hunk 1]
```diff
@@ -3755,6 +3771,21 @@ def test_recentactions_link(self):
...
+
+    def test_recentactions_delete(self):
+        response = self.client.get(reverse("admin:index"))
+        should_contain = '<span class="visually-hidden">Deleted:'
+        self.assertContains(response, should_contain)
+
+    def test_recentactions_added(self):
+        response = self.client.get(reverse("admin:index"))
+        should_contain = '<span class="visually-hidden">Added:'
+        self.assertContains(response, should_contain)
```
Comment (felixxm) [inline tests/admin_views/tests.py:3787]: I would move it to the single test method, e.g.
```suggestion
    def test_recentactions_description(self):
        response = self.client.get(reverse("admin:index"))
        for operation in ["Added", "Changed", "Deleted"]:
            with self.subTest(operation):
                self.assertContains(response, f'<span class="visually-hidden">{operation}:')
```

**Review by thibaudcolas — APPROVED** [review]
👍 all working as expected in desktop Safari VoiceOver.

**Comment by felixxm** [comment]
@nmenezes0 Do you have time to keep working on this? I only left reformatting suggestions.

**Comment by nmenezes0** [comment]
> @nmenezes0 Do you have time to keep working on this? I only left reformatting suggestions.

Yes, I have time to keep working on this today and will make the suggested changes.

**Comment by felixxm** [comment]
@nmenezes0 Thanks :+1: Welcome aboard :boat: You're a Django contributor no. 2400 :partying_face:

**Comment by felixxm** [comment]
buildbot, test on selenium.

**Pull request merged**